AVFoundation-Combine/Video Player/VideoPlayerViewController.swift

```diff
@@ -65,9 +72,51 @@ final class VideoPlayerViewController: UIViewController {
         videoPlayerContentOverlay.progressSlider.addTarget(self, action: #selector(onSliderThumbTouchedUp), for: .touchUpInside)
     }
     
-    // MARK: UI Actions
+    private func setupReplayControls() {
```

Comment:
Just as it was done with the other controls:

* Code dealing with view hierarchy, layouts etc should be placed in `VideoPlayerContentOverlay` as much as possible so that it is separated neatly and is not affecting the legibility of the Publisher code sample.
* As per "Best practices" any custom UI should be added to `avPlayerViewController.contentOverlayView` 
* I can definitely provide an asset for the replay Icon